vocode/streaming/models/transcriber.py

```diff
@@ -78,6 +80,8 @@ def from_input_device_config_experimental(
         endpointing_config: Optional[EndpointingConfig] = None,
         vad: Optional[bool] = False,
         experimental: Optional[bool] = False,
+        utterance_end_ms: Optional[float] = None,
```

Comment:
These need to be in the `from_input_device_config_vad` and `from_input_device_config_experimental`

Reply:
Should be fixed up now